pubsub/tests/unit/pubsub_v1/subscriber/test_message.py

```diff
@@ -135,7 +136,9 @@ def test_drop():
 
 def test_lease():
     msg = create_message(b"foo", ack_id="bogus_ack_id")
-    with mock.patch.object(msg._request_queue, "put") as put:
+
+    pytest_warns = pytest.warns(DeprecationWarning)
+    with pytest_warns, mock.patch.object(msg._request_queue, "put") as put:
         msg.lease()
         put.assert_called_once_with(
             requests.LeaseRequest(ack_id="bogus_ack_id", byte_size=30)
```

Comment:
Seems like we need an assertion that the warning is emitted.

Reply:
There is an assertion, `pytest_warns` (fails the test if a warning is not raised).

Reply:
No worries, then:  I didn't know it had a built-in assertion.